src/flame/train.py

```
@@ -755,6 +771,64 @@ def main(job_config: JobConfig):
             )
 
+            if torch.distributed.get_rank() == 0:
+                if job_config.checkpoint.enable_checkpoint:
+                    hf_target_path = None
+                    dcp_save_path = os.path.join(
+                        job_config.job.dump_folder,
+                        job_config.checkpoint.folder,
+                        f"step-{train_state.step}",
+                    )
+
+                    # TODO: Haven't tested this one yet
```

Pylint (3.3.7)

[warning] TODO: Haven't tested this one yet

(W0511)